features/yield/FarmingCalculator.tsx

```diff
@@ -60,15 +66,18 @@ const FarmingCalculator = () => {
     timeRemainingToFarmingRoll / 1000 / 60 / 60;
 
   // Yield inputs:
-  const [umaPrice, setUmaPrice] = useState<string>("");
+  const [umaPrice, setUmaPrice] = useState<number>(0);
```

Comment:
Why'd you change this?

Reply:
I thought it would be more consistent but it's fine to leave it as strings. I've reverted it.

Reply:
Yeah actually more consistent to leave like the others as ""